interface/usergroup/usergroup_admin.php

```diff
@@ -20,15 +20,17 @@
  * @link    http://www.open-emr.org
  */
 
-require_once("../globals.php");
-require_once("../../library/acl.inc");
-require_once("$srcdir/auth.inc");
+require_once "../globals.php";
+require_once "../../library/acl.inc";
+require_once "$srcdir/auth.inc";
+use Symfony\Component\HttpFoundation\Request;
+use services\UserService;
```

Comment:
According to PSR, shouldn't the use stuff go above the require_once and have a empty line below it?